README.md

```diff
@@ -39,6 +39,7 @@ Currently, DiffKemp runs on Linux and needs the following software installed:
 * Python 3 with CFFI (package `python3-cffi` in Fedora and Debian)
 * Python packages from `requirements.txt` (run `pip install -r requirements.txt`)
 * CScope (when comparing versions of the Linux kernel)
+* GoogleTest (gtest) for running the C++ tests
```

Comment:
Maybe add information that GTest can be vendored?
```suggestion
* GoogleTest (gtest) for running the C++ tests (can be vendored by using `-DVENDOR_GTEST=On` in CMake)
```

Reply:
Good idea, fixed